File: react/src/app/dto/NewGameDto.ts
export default class NewGameDto {

    private adminId: number;
    private name: string;
    private goals: string;
    private prizes: string;
    
    constructor(adminId: number, name: string, goals: string, prizes: string){
        this.adminId = adminId;
        this.name = name;
        this.goals = goals;
        this.prizes = prizes;
    }

    toJSON() {
        return {
            "adminId": this.adminId,
            "name": this.name,
            "goals": this.goals,
            "prizes": this.prizes
        }
    }

}

File: spring/game/src/main/java/com/gigilife/game/game/dto/NewGameDto.java
package com.gigilife.game.game.dto;


import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class NewGameDto {

    private Integer adminId;
    private String name;
    private String goals;
    private String prizes;

}